dex/src/main/scala/com/wavesplatform/dex/api/http/routes/MatcherApiRoute.scala

```diff
@@ -1174,6 +1178,31 @@ class MatcherApiRoute(
     }
   }
 
+  @Path("/debug/address/{address}")
+  @ApiOperation(
+    value = "Get state (balances, placement queue by address",
+    httpMethod = "GET",
+    authorizations = Array(new Authorization(SwaggerDocService.apiKeyDefinitionName)),
+    tags = Array("debug"),
+    response = classOf[HttpAddressActorState]
```

Comment:
`Actor` is a detail of implementation, let's remove this part

Reply:
done